docs/releasing.md

```diff
@@ -106,12 +55,9 @@ echo "Upload the package zip to CWS dev dashboard"
 # Select `lighthouse-4.X.X.zip`
 # _Publish_ at the bottom
 
-echo "Verify the npm package won't include unncessary files"
-npm pack --dry-run
-npx pkgfiles
-
-echo "ship it"
+# Publish to NPM
 npm publish
```

Comment:
do we know that we're def running this from the pristine location?

Reply:
oh well this is the manual instructions piece not 100% auto yet :) but yeah added a `cd` and comment 👍